### ISSUE TITLE
Copay no longer communicating with Trezor since latest firmware update

### ISSUE THREAD

**Issue by RustyNomad** [bitpay/copay#4642]
Not sure whether anybody else have experienced this same problem. Since the last firmware update (1.3.6) on Trezor my Copay wallet no longer communicates with it. When trying to create/import a hardware wallet I get the message 'Waiting for Trezor' which never goes away and there seems to be no communication with the Trezor even though its plugged in and working properly in myTrezor etc..

Using the Chrome extension of Copay ver 2.3.0

**Comment by matiu** [comment]
Thanks for the report, we will test it and give you feedback.

**Comment by JDonadio** [comment]
@RustyNomad I could reproduce it. I already found the bug and I'll send the fix in a few minutes. Thanks a lot for this report!

**Comment by JDonadio** [comment]
Ok, the new release for chrome was already send and will be available in a few minutes. Please if the problem remains let us know ASAP.

**Comment by RustyNomad** [comment]
@JDonadio That was super quick, will get it a little later and test, thanks for the fix.

**Comment by RustyNomad** [comment]
@JDonadio Updated to version 2.4.1 but still getting the 'Waiting for Trezor' message with no connection happening. Tried creating a Trezor wallet as well as importing a hardware wallet. Tried myTrezor afterwards and that works fine. Checked Chrome browser and its up to date (ver: 52.0.2743.82 m).

**Comment by JDonadio** [comment]
@RustyNomad can you check your trezor chrome extension? I'm using the 1.1.2 (latest version -  today)

**Comment by RustyNomad** [comment]
@JDonadio Yes also using ver 1.1.2, updates it earlier when it did not work at first. I removed both Copay and the Trezor extensions, restarted Chrome and added both back but still no joy.

**Comment by matiu** [comment]
Can you try disabling the password function? I think that was reported before.

**Comment by RustyNomad** [comment]
@matiu Disabled the password function but still the same.

**Comment by JDonadio** [comment]
@RustyNomad can you please do the next: Open Copay (ChromeApp) right-click > inspect and select console tab. Try to import a wallet and check the logs. Let me know if there is any errors there please

**Comment by RustyNomad** [comment]
@JDonadio It worked.

Could not do what you suggested, could not find any right click option/menu on the chrome app, unless I missed something. What I did was to startup Copay, I then went to the extensions in the chrome browser and clicked on the index.html link appearing at the Copay extension listing in the extension list.

This opened up the 'debug' windows where I could see the code for the html file. The communication with the Trezor device kicked in the moment I clicked that link. Everything ran fine and I was able to import the public keys from the device into Copay.

Not sure why it did not do it before. I removed Copay afterwards and restarted the browser and added Copay back and did a hw import again and it worked without any problems and without having to open the 'debug' window again. As said, not sure what caused this but seems like it might have been something with my browser which just needed a proverbial kick in the head.

Thanks for the assistance.
